docs/dev/developer-guides/bridging/l1-l2-interop.md

```diff
@@ -1,72 +1,76 @@
+If users wanted to deposit or withdraw funds, they had to send a transaction request to the smart contract. The request was appended to the priority queue. The queue had the following rules:
 
-- The operator must do at least `X` amount of work (see below) on the priority queue or the priority queue should be empty.
+1. All transactions were processed sequentially, i.e. FIFO.
+2. Each priority operation had to be processed by the operator within `X` days from submission to the contract.
 
-In other words, we require the operator to do its best instead of requiring a strict deadline. The measure of "the work" is still to be developed. Most likely it will be the number of `gas` the priority operations used.
+The first rule was strictly enforced by the smart contract. 
 
-In the future, we will also add the ability to "prioritize" L1->L2 transactions, allowing users to speed up the inclusion of their transactions in exchange for paying a higher fee to the operator.
+The second rule could be violated if the operator became malicious or unavailable. If that happened, the system entered **exodus mode**, where no new blocks were processed and users could withdraw their funds without operator interference.
 
-## Priority mode
+#### How it works in zkSync Era
 
-If the operator fails to process the needed L1 transactions, the system enters in 'Priority mode'. In this mode, everyone can become an operator by staking tokens. The exact details of the priority mode are still under development and will be described in more detail closer to the mainnet launch.
+The previous design was sufficient for a system supporting only deposit and full exit operations. 
 
-To reduce risks, the alpha mainnet will start with a mechanism to instantly stop and upgrade the network, which contradicts the purpose of the priority mode. Priority mode will be gradually introduced in the following releases.
+However, zkSync Era supports general smart contract computation which includes complex operations. For this reason, some of the principles had to evolve in order to preserve the stability of the network.
 
-## L2 -> L1 messaging
+1. All transactions types are now supported by the priority queue. 
 
-The [L2 -> L1 communication](./l2-l1.md), in contrast to L1 -> L2 communication, is based only on transferring the information, and not on the transaction execution on L1. It is a built-in feature, which is made up of two parts: sending a message from L2 and reading it on L1. The first is implemented as a call to an L2 system smart contract. And the second is implemented on the zkSync L1 smart contract as a getter function.
+    In Lite, users may have had their funds locked on an L2 smart contract, and not on their own L2 account. Now, before moving funds to L1, they can execute a transaction on the zkSync Era network which releases the funds from the smart contract so they can move them.
```

Comment:
Using, instead of in?
I'm curious now about the previous statement "The first is implemented as a call to an L2 system smart contract", was this also incorrect?
@StanislavBreadless clarification please?
Yes about the branding.

Reply:
ok, i removed the Lite content

Reply:
Please comment out the text, instead of deleting it, as we may use it in the future. It would be good to also have a short comment for future editors why this (commented out) bit is there.

Reply:
I would rather begin again and control the process tbh. The content is still available in git.

Reply:
gone now